### PR TITLE
Adding a periodic job for OKD/SCOS and set it as an acceptance job

### PR DESCRIPTION
**Pull request by aleskandro**
This PR adds a periodic job to test OKD/SCOS, as a blocker for its releases.
/cc @LorbusChris @vrutkovs @bradmwilliams 

Refers [OKD-6](https://issues.redhat.com//browse/OKD-6)

### CODE REVIEW

**Comment by aleskandro** [comment]
Blocked by https://github.com/openshift/ci-tools/pull/3105 and https://github.com/openshift/machine-config-operator/pull/3382

File: ci-operator/config/openshift/release/openshift-release-master__okd-scos-4.12.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      stream: okd-scos
+      version: "4.12"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-aws-ovn
+  interval: 48h
```
Comment (vrutkovs) [inline ci-operator/config/openshift/release/openshift-release-master__okd-scos-4.12.yaml:19]: not required - release-controller would start the job whenever necessary (on a new nightly)

**Review by vrutkovs — CHANGES_REQUESTED** [review]

File: ci-operator/config/openshift/release/openshift-release-master__okd-scos-4.12.yaml (hunk 1 above)
Comment (vrutkovs) [reply ci-operator/config/openshift/release/openshift-release-master__okd-scos-4.12.yaml:19]: Nevermind, its okay to have it

**Review by vrutkovs — APPROVED** [review]
/lgtm
/hold cancel

**Review by bradmwilliams — APPROVED** [review]

**Pull request merged**